docs/docs/databases/ms-sql-server/getting-started/cloud.mdx

```diff
@@ -59,17 +59,17 @@ Once you add the database, you'll see your database pop up on the sidebar.
 If you have existing tables, head to the database page by clicking on the database name on the sidebar. You should see a
 list of tables.
 
-<Thumbnail src='/img/databases/ms-sql-server/3-manage-mydb.png' alt='Manage my-db' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/manage-db_step-3_console_2.10.1.png' alt='Manage my-db' width='1000px' />
 
 Track tables selectively or all of them so that Hasura can introspect the tables and create the corresponding GraphQL
 schema.
 
-<Thumbnail src='/img/databases/ms-sql-server/4-track-tables.png' alt='Track tables' width='1000px' />
+<Thumbnail src='/img/databases/ms-sql-server/track-tables_step-3_console_2.10.1.png' alt='Track tables' width='1000px' />
```

Comment:
This is too wide. Please set the width so it's the same size as the actual UI. 


<img width="1725" alt="Screenshot 2022-10-18 at 14 26 29" src="https://user-images.githubusercontent.com/94021366/196442621-fc935ffb-3fa7-48ed-aa4f-44dddabeb32b.png">